dynaconf/loaders/__init__.py

```diff
@@ -263,7 +263,7 @@ def enable_external_loaders(obj):
             obj.LOADERS_FOR_DYNACONF.insert(0, loader)
 
 
-def write(filename, data, env=None):
+def write(filename, data, merge=False, env=None):
```

Comment:
I would add the `merge=False` as the last argument, because if someone is already relying on positional argument order it will break.

```suggestion
def write(filename, data, env=None, merge=False):
```

Reply:
I agree. I did that initially but changed it to reflect the exemplified use in the docs. I'll change.